src/Raven.Server/Documents/Handlers/Admin/AdminLogsHandler.cs

```diff
@@ -140,6 +134,9 @@ await using (var entryStream = entry.Open())
                         {
                             await DebugInfoPackageUtils.WriteExceptionAsZipEntryAsync(e, archive, fileName);
                         }
+
+                        if (hasLogDateTime && to.HasValue && logDateTime > to)
```

Comment:
Not sure I follow. Given logs:
14 Dec
15 Dec
18 Dec

and arguments: from = 16 Dec to = 17 Dec
what we will get?

'from' will filter out 14 and 15, but will not filter out 18th
but 'to' is processed after 18th was written so it will not have chance to filter it out?

Reply:
The list of log files does not account for breaks in time or missing files.
In the situation you're describing, there are three files with `LastWriteTime` values of `14 Dec`, `15 Dec`, `18 Dec`.

Thus, a pair of files with `LastWriteTime` values of `15 Dec` and `18 Dec` suggests that the logs `from = 16 Dec to = 17 Dec` should be in the file with `LastWriteTime = 18 Dec`.

We only select files that _should_ contain the requested information without considering their content.

Reply:
Yes, indeed, the situation with oranges and apples does occur, but this is a secondary issue, in addition to my original fix.